target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-cm520-79f.dts

```diff
@@ -218,95 +220,19 @@
 			#size-cells = <1>;
 
 			partition@0 {
-				label = "SBL1";
```

Comment:
NACK, what is the reason for dropping individual partitions and just blobing them up as "Bootloader" which is incorrect